lib/ui/components/captioned_text_widget.dart

```diff
@@ -0,0 +1,67 @@
+// leading icon can be passed without specifing size, this component sets size to 20x20
+import 'package:flutter/material.dart';
+import 'package:photos/ente_theme_data.dart';
+
+class CaptionedTextWidget extends StatelessWidget {
+  final String title;
+  final String? subTitle;
+  final TextStyle? textStyle;
+  final bool makeTextBold;
+  final Color? textColor;
+  const CaptionedTextWidget({
+    required this.title,
+    this.subTitle,
+    this.textStyle,
+    this.makeTextBold = false,
+    this.textColor,
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final enteTheme = Theme.of(context).colorScheme.enteTheme;
```

Comment:
I remember suggesting this pattern.. But may be we can create two vars here to reduce the verbosity in the remaining code..
```
final enteTextStyle
final enteColorSceme
```

Edit: After adding above comment, just discovered you are already similar stuff :)